src/i18n/resources/en/common.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   add: 'Add',
   apply: 'Apply',
   areYouSureYouWantToDeleteAllTableData: 'Are you sure you want to delete all data from the table?',
+  areYouSure: 'Are you sure?',
```

Comment:
I know, I wanted to create a more generic message that could be used elsewhere, any thoughts?

`Are you sure you want to continue?`
`Are you sure you want to ...`?

Let me know if you can find a generic confirmation, otherwise let's use what you wrote here!

Reply:
Sure, let's do so